subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/checks/Checks.kt

```diff
@@ -58,6 +59,8 @@ interface Checks : LabelChecks {
 
     fun isNotVisible()
 
+    fun hasVisibility(visibility: ViewMatchers.Visibility)
```

Comment:
I was confused that we have `isNotVisible` and `hasVisibility`. We should have one API to do one thing

Reply:
I think the current setup adds confusion for users and I don't see a value for having `hasVisibility` fun

Reply:
> can't delete the old ones, because it will break backward compatibility.

For the record, changing API is possible and not a blocker. 
We only need to do it in multiple steps with deprecation and changing clients.